gitlabform/gitlabform/core.py

```diff
@@ -411,9 +410,20 @@ def process_secret_variables(self, project_and_group, configuration):
             return
 
         logging.debug("Secret variables BEFORE: %s", self.gl.get_secret_variables(project_and_group))
+
         for secret_variable in sorted(configuration['secret_variables']):
             logging.info("Setting secret variable: %s", secret_variable)
```

Comment:
If we leave this line here then for a secret variable that is going to be deleted:
* in verbose mode you will only get a log output saying that it is going to be set,
* in debug mode you will get a log output saying that it is going to be set and then a line saying that is going to be deleted...

This will look misleading.

Therefore I suggest moving this line just above the code that actually does set/create the secret variable and changing the level in line 420 to `info`.

Reply:
This applies to the code in group variables function too.